src/codeu/chat/client/commandline/Chat.java

```diff
@@ -108,6 +109,8 @@ public void invoke(Scanner args) {
         System.out.println("  u-sign-in <name>");
         System.out.println("    Sign in as the user with the given name.");
         System.out.println("  exit");
+        System.out.println("  version check");
+        System.out.println("  Display the version of the server.");
```

Comment:
Updated the styling by adding the indent and also put the command before the exit command